### PR TITLE
readme gramatical changes

### PR DESCRIPTION
**Pull request by byronbytes**
Updated some things in the README so it's more gramatical
Also I really like these, and I use them on my profile (I even have the secret trophy!)

### CODE REVIEW

**Comment by byronbytes** [comment]
> Someone is attempting to deploy a commit to a Personal Account owned by @ryo-ma on [Vercel](https://vercel.com).
> 
> @ryo-ma first needs to [authorize it](https://vercel.com/git/authorize?type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%22d23d9c29e966b7d593a0b5e98999db216c1263df%22%7D%2C%22id%22%3A%22QmaKmSW18BpoTJCcUc2YVZPWTfBRZPndvY3r2m7SHbD4LK%22%2C%22org%22%3A%22ryo-ma%22%2C%22prId%22%3A102%2C%22repo%22%3A%22github-profile-trophy%22%7D).

What does this mean? And also I can't create an account right now, but I will later.

File: README.md [hunk 1]
```diff
@@ -64,10 +64,10 @@ Ranks are `SSS` `SS` `S` `AAA` `AA` `A` `B` `C` `UNKNOWN` `SECRET`.
...
 | ---- | ---- |
 |  SSS, SS, S  | You are at a hard to reach rank. You can brag.  |
-|  AAA, AA, A  | You will reach the rank if you do your best. Let's aim here first.  |
-|  B, C  | You are in a growing process.  |
-| UNKNOWN | You have not yet taken action. Let's act first. |
-| SECRET | The rank is very rare. The trophy will not be displayed until the conditions are met. |
+|  AAA, AA, A  | You will reach this rank if you do your best. Let's aim here first.  |
+|  B, C  | You are currently making good process. Let's aim a bit higher.  |
+| UNKNOWN | You have not taken action yet. Let's act first. |
+| SECRET | This rank is very rare. The trophy will not be displayed until the conditions are met. |
```
Comment (spenserblack) [inline README.md:70]:
```suggestion
| SECRET | This rank is very rare. The trophy will not be displayed until certain conditions are met. |
```

File: README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@
 
 # Quick Start
 
-Add following code to your readme.  
+Add the following code to your readme.  
 Change the `?username=` value to your GitHub's username.
```
Comment (spenserblack) [inline README.md:37]: Perhaps these can be combined into one sentence.

**Review by spenserblack — COMMENTED** [review]
Since grammatical change are being made, maybe these can be changed to.

Also, I couldn't highlight it in this PR right now (guess I can only comment on changed lines or lines *near* changes), but there's a part that says
>Add optional parameter of theme.

That could probably be changed to more natural English.

**Comment by spenserblack** [comment]
>What does this mean? And also I can't create an account right now, but I will later.

You don't need to make an account. What Vercel does is create a preview URL for each pull request so that maintainers can see what the new version would effectively look like. It's important that the owner(s) approve deployment first, because someone could theoretically do some bad stuff like
```typescript
// index.ts
export default async(req: ServerRequest) => {
  req.respond({ body: Deno.env.get("SECRET") });
};

File: README.md (hunk 2 above)
Comment (byronbytes) [reply README.md]: Updated so it is combined to one sentence.

**Comment by ryo-ma** [comment]
LGTM

**Review by ryo-ma — APPROVED** [review]

**Pull request merged**